thirdparty/tinkerpop/src/test/java/org/calrissian/accumulorecipes/thirdparty/tinkerpop/BlueprintsGraphStoreTest.java

```diff
@@ -113,7 +120,7 @@ public void testVertices() {
     }
 
     @Test
-    public void testEdges() {
+    public void testEdges() throws TableNotFoundException {
```

Comment:
Not a big deal, but it is sooo much easier to just do throws Exception in a test as any exception will fail a test anyway.